src/meson.build

```diff
@@ -0,0 +1,17 @@
+raylib_src = [
+  'raudio.c',
+  'rcore.c',
+  'rglfw.c',
+  'rmodels.c',
+  'rshapes.c',
+  'rtext.c',
+  'rtextures.c',
+  'utiles.c'
+]
+
+ray_lib = static_library('raylib,
```

Comment:
This should just be a regular `library()`, and respect Meson's "default_library" option for determining whether to build a static or shared library.

Cmake has an option to control which one to build (it defaults to off, in Meson this would be setting `default_options: ['default_library=static']`).